conf/modules/methylation.config

```diff
@@ -38,7 +38,7 @@ process {
         ext.args = '--combine-mods --cpg --combine-strands --partition-tag HP'
 
         publishDir = [
-            path: { "${params.outdir}/methylation_pileups/modkit/pileup/${meta.id}" },
+            path: { "${params.outdir}/methylation_pileups/modkit/pileup/phased/${meta.id}" },
```

Comment:
Just a comment, when we implement this pipeline at CG we would have to change the name of the output files. Either here in the pipeline or when we copy the output files to the housekeeper bundle for storage.

Reply:
Might as well be more explicit with naming here.